RevenueCatUI/PaywallView.swift

```diff
@@ -8,29 +8,38 @@ public struct PaywallView: View {
     private let offering: Offering
```

Comment:
I feel like we might have to rethink the naming for these at one point to minimize collisions with paywalls a customer might already have. 

Like, say you have a paywall in place and are looking to migrate into ours, it'd suck if you have to rename or remove the one you already have right away in order to test out our SDK

Reply:
Good point. I'd say before we merge this and ship it, we can go through the entire public API (won't be much) and make sure it's solid and documentation is good.